app/main.py

```diff
@@ -1,3 +1,5 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    small_phrase = phrase.lower()
+    small_letter = letter.lower()
+
```

Comment:
There is an unnecessary empty line here. While it doesn't affect functionality, you might want to remove it for cleaner code.